### DISCUSSION TITLE
Regarding Wave output values after running shel.nml

### DISCUSSION THREAD

**Discussion by krishnanathira** [NOAA-EMC/WW3#806] [Q&A] [answered]
Hi

I have used shel.nml file to run my setup. The output values (say "hs"), show very large values in the range of 50-60 after a few timesteps of simulation. The initial few timesteps are showing normal ranges. Later it shows very large values. Anybody, please suggest to me what can be the reason behind this. 

Many thanks!

**Comment by aliabdolali** [comment, marked as answer]
@krishnanathira Have you checked your time steps according to [CFL criteria](https://github.com/NOAA-EMC/WW3/wiki/FAQs-page#cfl-condition)?

**Reply by krishnanathira** [reply]
Thanks for the quick response.
My wind data is of 3 hours duration and I have selected timesteps for the simulation as follows
&TIMESTEPS_NML
  TIMESTEPS%DTMAX        =  10800.
  TIMESTEPS%DTXY         =  3600.
  TIMESTEPS%DTKTH        =  3600.
  TIMESTEPS%DTMIN        =  30.
/
According to the mentioned link above, 
The maximum allowable time step for spatial propagation is given by:
Δt=123766 * Δx * cosd(max(latitude))*f
I have calculated with Δx=0.25 degree, max(latitude)=55N, and f=0.04118 and Δt = 28.193

Can you please suggest to me if there is any error?
Thanks

**Reply by aliabdolali** [reply]
123766 * 0.25* cosd(55)*0.04118=730. How did your calculation end up with 28?

**Reply by krishnanathira** [reply]
Sorry! That was an error that happened when I calculated in the google calculator. You're right,  thanks for correcting it. I have got it as same as 730.8344 when calculated in a scientific calculator. 
Thanks

**Reply by krishnanathira** [reply]
As per the CFL criteria, 

TIMESTEPS%DTMAX        =  10800.
  TIMESTEPS%DTXY         =  720.
  TIMESTEPS%DTKTH        =  1440.
  TIMESTEPS%DTMIN        =  30.

Can you please suggest to me whether this works fine??
Thanks

**Reply by aliabdolali** [reply]
I would use TIMESTEPS%DTMAX < 3*720.

**Reply by krishnanathira** [reply]
Thank you!
